### ISSUE TITLE
Feature: Automate getting started guide

### ISSUE THREAD

**Issue by bernardhalas** [berops/claudie#1228]
### Motivation
When preparing for a new Claudie demo on new project, creating the accounts is susceptible to copy/paste mistakes and takes a substantial amount of time. Even when not using DNS and LBs, I need to account for ~2 hrs of time per one cloud provider and the result is not guaranteed. The approach of copy/pasting 10-11 commands per cloud provider has the following risks:
- copy/paste errors (e.g. in case of a different shell environment - `fish` or `zsh`)
- mix 'n match of various naming conventions (e.g. `userocid` and `user_ocid`) - difficult to check the syntax of the commands
- rewiring the client CLIs like `oci` `aws` or `gcloud` especially if a user already has them bound to different accounts/projects
- various python versions + installations which might prevent a successful CLI installation
- copy/paste errors in the input manifest

### Description
I'd expect to have an interactive command-line CLI, that would:
- ask _only_ the absolutely essential information
- fire up provider CLI (from an image) and do the login
- perform all the actions and generate secrets locally
- apply the claudie manifest + secrets + basic input manifest which the user can tune later

### Exit criteria

- [ ] High-level design is reviewed and OK'd by the team
- [ ] The CLI is implemented
- [ ] Docs are updated

**Comment by Despire** [comment]
> fire up provider CLI (from an image) and do the login

This does not make much sense, would we spin up multiple images each for a unique provider ?

Why not just make it work directly with the CLIs (if possible) without images. With this the CLI/scripts could be run in a single docker container (possibly if the CLI/scripts would also install the necessary dependent cloud CLI's if not present)

**Comment by bernardhalas** [comment]
I agree with the statement @Despire . I suggest to pack the resulting CLI in a docker image with provider CLIs.
